config/locales/views/circuitverse/ar.yml

```
@@ -0,0 +1,154 @@
+        description4_html: "2.4 إذا كنت تعتقد أن حسابك لم يعد آمنًا (على سبيل المثال، إذا فقدت كلمة المرور أو سُرقت أو تم الكشف عنها دون إذنك)، فيجب عليك تغيير كلمة المرور فورًا. إذا لم تتمكن من الوصول إلى حسابك لتغيير كلمة المرور، يُرجى إبلاغنا عبر البريد الإلكتروني <a href='mailto:support@CircuitVerse.org'>support@CircuitVerse.org</a>."
+        usage_rule:
+          main_heading: "3. قواعد الاستخدام"
+        description1: "3.1 يدعم فريق CircuitVerse حرية التعبير. ومع ذلك، تم تصميم CircuitVerse لجمهور واسع، وقد يكون بعض المحتوى غير مناسب لمجتمع CircuitVerse. لا يجوز لك استخدام خدمات CircuitVerse بأي شكل من الأشكال التي:"
+        list1_item1: "تعزز التحيز أو التمييز أو الكراهية أو العنف ضد أي فرد أو مجموعة؛"
+        list1_item2: "يهدد أو يضايق أو يخيف أي شخص آخر، سواء كان ذلك الشخص مستخدمًا لـ CircuitVerse أم لا؛"
+        list1_item3: "يحتوي على لغة فاحشة أو هجوم شخصي؛"
+        list1_item4: "يحتوي على محتوى جنسي صريح أو مواد تصويرية عنيفة؛"
+        list1_item5: "يوفر تعليمات حول كيفية القيام بأنشطة غير قانونية أو الحصول على منتجات غير مشروعة؛"
+        list1_item6: "يطلب من أي مستخدم معلومات تحديد الهوية الشخصية أو معلومات الاتصال أو كلمات المرور، باستثناء ما يتعلق بتنظيم فعاليات CircuitVerse اليومية؛ أو"
+        list1_item7: "يكشف عن معلومات تحديد الهوية الشخصية أو معلومات الاتصال أو كلمات المرور لأي شخص آخر دون إذنه."
+        description2: "3.2 عند استخدامك لـ CircuitVerse، فإنك توافق على الامتثال لجميع القوانين واللوائح المعمول بها. لا يجوز لك استخدام CircuitVerse بأي طريقة غير قانونية، بما في ذلك مضايقة أو ملاحقة أو تشويه سمعة أي شخص آخر."
```

YAMLlint (1.35.1)

[error] duplication of key "description2" in mapping

(key-duplicates)